tests/layer_tests/pytorch_tests/test_pow.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from pytorch_layer_test_class import PytorchLayerTest
+import numpy as np
+import torch
+
+
+@pytest.mark.parametrize('test_input', [(np.array([[1, 2], [3, 4]], dtype=np.float32),
+                                         np.array([[1, 1], [2, 2]], dtype=np.float32),),
+                                        (np.array([[1, 2], [3, 4]], dtype=np.float32),
+                                         np.array([2, 3], dtype=np.float32),),
+                                        (np.array([[1, 2], [3, 4]], dtype=np.float32),
+                                         np.array([2], dtype=np.float32),),
+                                        (np.array([5, 6], dtype=np.float32),
+                                         np.array([[1, 2], [3, 4]], dtype=np.float32),),
+                                        (np.array([5], dtype=np.float32),
+                                         np.array([[1, 2], [3, 4]], dtype=np.float32),)])
```

Comment:
When we cover more data types (if any, fp16 for example, I don't know), will we just double this block? Looks like for all operations that support broadcasting we will do this "dimensional exercise". Can we provide some helpers to produce less boilerplate code in each test? @mvafin?

Reply:
I don't think we should double this block with additional types. We want to check translation correctness, not correctness of openvino Power operation